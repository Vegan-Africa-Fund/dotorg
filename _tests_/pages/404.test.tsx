export {}
import React from 'react';
import renderer from 'react-test-renderer';
import Custom404 from '../../pages/404';

test("Error page is rendered correctly", () => {
 const errorPage = renderer.create(<Custom404 />).toJSON()
 expect(errorPage).toMatchSnapshot()
})