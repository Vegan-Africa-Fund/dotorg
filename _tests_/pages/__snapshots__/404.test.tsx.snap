// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error page is rendered correctly 1`] = `
<div
  className="MuiBox-root MuiBox-root-1 page"
>
  <h1>
    404 | Page Not Found.
  </h1>
  <button
    className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary"
    disabled={false}
    onBlur={[Function]}
    onClick={[Function]}
    onDragLeave={[Function]}
    onFocus={[Function]}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onMouseDown={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onMouseUp={[Function]}
    onTouchEnd={[Function]}
    onTouchMove={[Function]}
    onTouchStart={[Function]}
    tabIndex={0}
    type="button"
  >
    <span
      className="MuiButton-label"
    >
      Go home
    </span>
  </button>
</div>
`;
